python/tests/core/test_great_expectation_engine.py

```diff
@@ -13,13 +13,21 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 #
+import importlib.util
 
-import great_expectations as ge
 import hsfs.expectation_suite as es
+import pandas as pd
+import pytest
 from hsfs import feature_group, validation_report
 from hsfs.core import great_expectation_engine
 
 
+HAS_GREAT_EXPECTATIONS = False
+if importlib.util.find_spec("great_expectations"):
+    HAS_GREAT_EXPECTATIONS = True
+    import great_expectations.core
```

Comment:
Is there a reason not to use hsfs.core.constans.HAS_GREAT_EXPECTATIONS here?

Reply:
Ah nej sorry, forgot to extend the search to the tests folders